Add page about in-VR menus in NanoVer iMD

rhoslynroebuck opened the pull request:
Updated the page about the in-VR menus in NanoVer iMD. Split this into the following sections:
* **Using your VR controllers** - including diagrams of the buttons that users will need to know to use NanoVer iMD and an explanation about how to click menu buttons in VR.
* **Main menu**: explanation of the different options for connecting to a server with videos & text explaining how to do each one. 
* **In-simulation controls**: separated this section into (a) main controls, (b) right hand controls, and (c) left hand controls. 

These are the sub headings for the in-sim controls:

* **Main controls**: these are the controls that are accesible when no menu is open. This includes interacting with the molecule and moving/rescaling the box (+videos and text explanations)
* **Right hand controls**: changing the force scaling and interaction type (+videos and text explanations)
* **Left hand controls**: running simulation commands, changing simulation, and customizing your avatar (+videos and text explanations)

It's probably not the best page ever, but we don't know if we are going to be changing the UI soon. So the aim with this PR is to get something in quickly that has enough information for a new user to get started. Please do feel free to make suggestions and/or make edits!

rhoslynroebuck commented:
Just something to add, the version of NanoVer iMD that this page documents is not actually the latest version. There is a newer version with only one difference to the UI (as far as I know), which is that there is a button to "step" frames. However, I couldn't find an apk for this version on the git repo, and I need an apk to create good quality videos. As this is not a huge difference, I just used the old apk and didn't document this button. I'm also not sure if this button will be there permanently, as it only makes sense for recordings, not live simulations. 

I think Mark will be working on automatically generating the apk with each new build, so hopefully this kind of issue won't be a problem in the near future.

hjstroud reviewed (changes requested):
We have reviewed this PR in person, and have subsequently added and committed many of the minor changes suggested. 

It would be worth splitting the separate headings on this page into nested pages that a user can click through to each section individually, and make the information more digestible/navigable. In addition, once these are reformatted, adding a short description of each page's contents on the "main page" for the vr menus will help direct people to the information they require. 

Looking good, not too much to change!

Mohamed-Dhouioui commented on requirements.txt:
```diff
@@ -1,4 +1,5 @@
-sphinx-design
\ No newline at end of file
+sphinx-design
+sphinxcontrib-video
```
When using this package `sphinxcontrib-video` I got some warnings of deprecated modules like `sphinx.util.SkipProgressMessage` and `sphinx.util.progress_message`. I don't know if other people got the same warnings but I thought it is worth mentioning here in case something needs to be changed in the future.

Mohamed-Dhouioui replied:
I just created a new environment and a clean build, and I no longer get this warning.

Mohamed-Dhouioui reviewed (commented):
Other than warnings that I got when using `sphinxcontrib-video`, it seems to me that having in-vr menus over three separate pages is a bit too extensive and too deep in the documentation(e.g the page for **Using your VR controllers** is not that long and can be included in one of the others). I believe that for someone who is just starting to use NanoVer and wants to get started with understanding menus, navigating to `home/Tutorials/Nanover imD/In-VR menus/ Main menu: connecting to a NanoVer server` is a bit cumbersome and more so when this is not indexed in the left navigation menu. Maybe a simpler way is to have them indexed in the left navigation menu and reorganize them into two pages.

rhoslynroebuck commented:
OK, thanks for the comments both! I have done a rehaul to take into account what you have both said. 

I agree that there is a lot of information to present to the user on a single page. I tried nesting the pages as you both suggested, but the links to these pages just got lost in the whole page and I found it was difficult to navigate the page. 

So then I tried having all of the information on one page, but breaking it up into sections with drop downs, forced line breaks, horizontal lines, and clearer heading names. I also added a section on "A typical iMD-VR workflow" to give step-by-step instructions to a new user on a basic workflow. I think this is the clearest layout of any that I have tried, but please do suggest any improvements! 

Could you guys check review the new changes, focusing on the general layout and structure? Please do feel free to look at the details if you want, but we can check these later on once we're happy with the structure.

Mohamed-Dhouioui reviewed (approved):
This looks better and easier to navigate. Great job Rhos! 💪🏽

hjstroud reviewed (commented):
I think the structure of this page is looking better, I like the use of dropdown menus to keep things looking neater. One comment I have is that I don't think there should be horizontal lines within a subsection, as it seems to make it less clear what is a separate section and what is a subsection. I haven't looked at the content in detail yet though!

rhoslynroebuck commented:
OK, thanks again for the reviews! I have finalised the structure based on the above comments & in-person discussions. 

My last commits have changed quite a lot of things (see the individual commit messages for details). I have added some images to section the page more easily, removed horizontal lines between subheadings, and formatted the headings/subheadings/line breaks/horizontal lines consistently across the entire page.

Mohamed-Dhouioui commented on source/highlights.rst:
```diff
@@ -56,7 +56,7 @@ Develop with NanoVer
-* `Get the source code of the VR client <https://github.com/IRL2/nanover-imd>`_
+* `Get the source code of the VR client <https://github.com/IRL2/nanover-imd-vr>`_
 
 NanoVer is distributed under the `MIT <https://github.com/IRL2/nanover-protocol/blob/main/LICENSE>`_ license.
```
```suggestion
NanoVer is distributed under the `MIT <https://github.com/IRL2/nanover-server-py/blob/main/LICENSE>`_ license.
```

rhoslynroebuck replied:
This change is also being addressed in https://github.com/IRL2/nanover-docs/pull/132 and is out of scope of this PR, so I'm gonna reject it here

Mohamed-Dhouioui reviewed (approved):
I went through this page to verify that all videos play okay and their content is clear. I think Rhos has done an excellent job documenting all the different menus and interfaces. I added a couple of suggestions on fixing links that got relocated with the recent repos renaming. Other than that, I am happy with how this looks.

rhoslynroebuck commented on source/highlights.rst:
```diff
@@ -56,7 +56,7 @@ Develop with NanoVer
 **Contribute to the code** or **develop your own custom applications**.
 
 * `Get the source code for the server <https://github.com/IRL2/nanover-protocol>`_
```
> 
Hey @Mohamed-Dhouioui, this change is being addressed in #132 and is out of scope of this PR, so I'm gonna reject it here
